### PR TITLE
add "AuthParams" to email and sms connection options

### PR DESCRIPTION
**Pull request by yinzara**

### CODE REVIEW

File: management/connection.go [hunk 1]
```diff
@@ -541,7 +546,7 @@ func (c *ConnectionOptionsAzureAD) Scopes() []string {
 }
 
 func (c *ConnectionOptionsAzureAD) SetScopes(enable bool, scopes ...string) {
-	tag.SetScopes(c, true, scopes...)
+	tag.SetScopes(c, enable, scopes...)
```
Comment (alexkappa) [inline management/connection.go]: Ouch! Sorry about that 😞

File: management/connection.go [hunk 2]
```diff
@@ -403,6 +406,8 @@ type ConnectionOptionsSMS struct {
 
 	OTP *ConnectionOptionsOTP `json:"totp,omitempty"`
 
+	AuthParams map[string]string `json:"authParams,omitempty"`
```
Comment (alexkappa) [inline management/connection.go]: How did you find out about this property? I have modeled the `ConnectionOptionsEmail` and `ConnectionOptionsSMS` connections by reverse engineering the dashboard... Is there any documentation on this?

Comment (yinzara) [reply management/connection.go]: Only way to find this was to actually configure the Passwordless Email and SMS connections and then call the Management API to see what comes back.

Comment (alexkappa) [reply management/connection.go]: Ah, I see! Thanks for clarifying that.

**Comment by yinzara** [comment]
Mind making a release so I can use this in the terraform provider? :)

**Comment by alexkappa** [comment]
Sure thing, here you go: https://github.com/go-auth0/auth0/releases/tag/v4.0.2

**Pull request merged**